README.md

```diff
@@ -32,11 +32,24 @@ Brief instructions to build for linux(Todo add other platforms) we will add deta
 
 Install eth or geth, npm and node.js (see https://docs.npmjs.com/getting-started/installing-node), then do:
 
-    git clone https://github.com/ethereum/remix
-    cd remix
-    npm install && npm run build && npm run start_node
+    * `git clone https://github.com/ethereum/remix`
+    * `cd remix`
+    * `npm install`
+    * `npm start`
+
+open `remix/index.html` in your browser.
+
+## TEST:
+
+For unit tests run `npm test`
 
-open remix/index.html in your browser.
+For local headless browser tests run `npm run test-browser`
+
+## DEVELOPING:
+
+Run `npm run start_dev` and open `http://10.5.50.211:8080` in your browser.
```

Comment:
oh, yes it prints that to the terminal, but it could just be `localhost` or something. i can fix that. Not sure if it will always be `http://10.5.50.211:8080` for everyone